content/v2.0/reference/syntax/delete-predicate.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Delete predicate syntax
+list_title: Delete predicate
+description: >
+  The InfluxDB `/delete` endpoint uses a SQL-like predicate syntax to determine
+  what data points to delete.
+menu:
+  v2_0_ref:
+    parent: Syntax
+    name: Delete predicate
+weight: 104
+v2.0/tags: [syntax, delete]
+related:
+  - /v2.0/reference/cli/influx/delete/
+---
+
+The InfluxDB `/delete` endpoint uses a SQL-like predicate syntax to determine
+what data [points](/v2.0/reference/glossary/#point) to delete.
+InfluxDB uses the delete predicate to evaluate each point in the time range
+specified in the delete request.
+Points that evaluate to `true` are deleted.
```

Comment:
I don't have a strong opinion on this but the predicate applies to the _series_ and timestamp only, not the value of fields on the point. So in that sense I wonder if it's clearer to say something like:

> Points within the specified time range who have series keys that evaluate to true for a given predicate are deleted.